homeassistant/core.py

```diff
@@ -583,6 +583,15 @@ def async_add_job(
         target: target to call.
         args: parameters for method to call.
         """
+        # late import to avoid circular imports
+        from .helpers import frame  # pylint: disable=import-outside-toplevel
+
+        frame.report(
+            "async_add_job is deprecated and will be removed in Home Assistant 2025.4; "
```

Comment:
This will log a message like the following, right?

> Detected that integration 'blabla' async_add_job is deprecated and will be removed...

That doesn't sound very good.

Reply:
Yeah thats not so great

Reply:
Adjusted, and linked to the blog post instead. That will make it easier to update later if devs have more question, or we need to add more helpful hints